client/src/components/AddPerk.svelte

```diff
@@ -0,0 +1,103 @@
+<script>
+    import { onMount } from 'svelte';
+    import { Link } from "svelte-routing";
+
+
+        // Form data for the new perk
+        let perk = {
```

Comment:
Can you check that this aligns with all the properties that Caleb has in his Perk.Svelte